aws/required-permissions.md

```diff
@@ -129,3 +131,82 @@ Check out the templates directly for more information.
+| `lambda:GetEventSourceMapping` | Action | Allow Cloud One File Storage Security to describe the event source mapping of Lambda functions in the managed account scanner stack. |
+| `lambda:GetFunction` | Action | Allow Cloud One File Storage Security to describe the Lambda functions in the managed account scanner stack. |
+| `lambda:GetFunctionConcurrency` | Action | Allow Cloud One File Storage Security to describe the concurrency configuration of the Lambda functions in the managed account scanner stack. |
+| `lambda:GetFunctionConfiguration` | Action | Allow Cloud One File Storage Security to describe the configuration of Lambda functions in the managed account scanner stack. |
+| `lambda:GetLayerVersion` | Action | Allow Cloud One File Storage Security to describe the information of a version of the Lambda layers created or managed by Cloud One File Storage Security. |
+| `lambda:GetProvisionedConcurrencyConfig` | Action | Allow Cloud One File Storage to describe the provisioned concurrency configuration of a Lambda function in the managed account scanner stack. |
+| `lambda:ListAliases` | Action | Allow Cloud One File Storage Security to list the aliases of Lambda functions in the managed account scanner stack. |
+| `lambda:ListProvisionedConcurrencyConfigs` | Action | Allow Cloud One File Storage Security to list the provisioned concurrency configurations of Lambda functions in the managed account scanner stack. |
+| `lambda:ListVersionsByFunction` | Action | Allow Cloud One File Storage Security to list the versions of the Lambda layers of Lambda functions in the managed account scanner stack. |
+| `lambda:PublishVersion` | Action | Allow Cloud One File Storage Security to publish a new version of the Lambda functions in the managed account scanner stack. |
+| `lambda:UpdateFunctionCode` | Action | Allow Cloud One File Storage Security to update the code of the Lambda functions in the managed account scanner stack. |
+| `lambda:UpdateAlias` | Action | Allow Cloud One File Storage Security to update the alias of Lambda functions in the managed account scanner stack. |
+| `lambda:UpdateFunctionConfiguration` | Action | Allow Cloud One File Storage Security to update the configuration of Lambda functions in the managed account scanner stack. |
+| `logs:DescribeLogStreams` | Action | Allow Cloud One File Storage Security to describe the log streams of the Lambda functions in the managed account scanner stack. |
+| `logs:FilterLogEvents` | Action | Allow Cloud One File Storage Security to filter the log events of the Lambda functions in the managed account scanner stack. |
+| `logs:GetLogEvents` | Action | Allow Cloud One File Storage Security to get the log events of the Lambda functions in the managed account scanner stack. |
+| `logs:GetQueryResults` | Action | Allow Cloud One File Storage Security to get the log query results of the Lambda functions in the managed account scanner stack. |
```

Comment:
```suggestion
| `logs:GetQueryResults` | Action | Allows Cloud One File Storage Security to get the log query results of the Lambda functions in the managed account scanner stack |
```
The third cell in this row contains a clause phrase, not a full sentence, so doesn't require a period. The verb form should be the same for each clause in this column. Because the implied subject (the managed policy) is singular, the verb should be singular which typically ends in "s".